dev-tools/make/mage.mk

```diff
@@ -1,13 +1,69 @@
-MAGE_VERSION     ?= v1.9.0
-MAGE_PRESENT     := $(shell mage --version 2> /dev/null | grep $(MAGE_VERSION))
-MAGE_IMPORT_PATH ?= github.com/magefile/mage
-export MAGE_IMPORT_PATH
-
-.PHONY: mage
-mage:
-ifndef MAGE_PRESENT
-	@echo Installing mage $(MAGE_VERSION) from vendor dir.
-	@go install -mod=vendor -ldflags="-X $(MAGE_IMPORT_PATH)/mage.gitTag=$(MAGE_VERSION)" ${MAGE_IMPORT_PATH}/...
-	@-mage -clean
-endif
-	@true
+# This is a minimal Makefile for Beats that are built with Mage. Its only
```

Comment:
Why are you adding new targets to the file as opposed to directly invoking `mage` from Travis without any shim? Is it because the Travis file isn't setup to do that?

Reply:
The goes with my comment above. This is really just `xpack.mk` renamed to `mage.mk`. These targets are the same targets shared with the other x-pack beats. The travis file does use these a lot.

If we want to remove the use of a Makefile shim completely, then I think we should move that to a different PR. This is currently trying to make `metricbeat` just like `x-pack/metricbeat`.

Reply:
The reason I asked was that `unit-tests` and `system-tests` were not in the former xpack.mk. But I see that this file is now being used to shim targets used from `metricbeat/Makefile`. So no problem. We'll be able to remove this shim makefile soon.